pegasus/forms/census.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './form'
+
+class Census2017 < Form
+  def self.normalize(data)
+    result = {}
+
+    result[:email_s] = required data[:email_s]
+    result[:name_s] = data[:name_s]
```

Comment:
because this is a text field that is entered by the user, this is better as `stripped data[:name_s]`, or `required stripped data[:name_s]` if it's required.